src/qos_net/Cargo.toml

```diff
@@ -12,8 +12,9 @@ borsh = { version = "1.0", features = [
     "derive",
 ], default-features = false }
 serde = { version = "1", features = ["derive"], default-features = false }
-hickory-resolver = { version = "0.24.1", features = [
-    "tokio-runtime",
+
+hickory-resolver = { version = "0.25.1", features = [
+    "tokio",
```

Comment:
feature was renamed to `tokio` as of https://github.com/hickory-dns/hickory-dns/releases/tag/v0.25.0